src/v2/Apps/MeetTheSpecialists/MeetTheSpecialistsApp.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react"
+
+interface MeetTheSpecialistsAppProps {}
+
+export const MeetTheSpecialistsApp: React.FC<MeetTheSpecialistsAppProps> = ({
+  children,
+}) => {
+  return <>{children}</>
```

Comment:
When ready we'll also want to use our `<MetaTags>` component for page title, etc